src/local/butler/lint.py

```diff
@@ -200,6 +196,13 @@ def execute(_):
     _execute_command_and_track_error('pylint ' + file_path)
     _execute_command_and_track_error('yapf -d ' + file_path)
 
+    py_import_order(file_path)
+    py_test_init_check(file_path)
+
+    # Skip files that cause futurize to report false positive issues.
+    if file_path.endswith('ndb_patcher.py'):
```

Comment:
This feels pretty hacky but we won't need to keep it around too long. After the migration we'll remove the futurize checks.

Reply:
SGTM. let's add a TODO here so we don't forget?

Reply:
+1 TODO with "python 3" word in it.